configuration/rules-blockly-items-things.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: documentation
+title: Rules Blockly - Items & Things
+---
+
+# Items &  Things
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+*Items* and *Things* are the [major entities of openHAB](https://www.openhab.org/docs/concepts/) to control and monitor the home. These can be accessed via the "Items & Things" section of the [Blockly Toolbox]({{base}}/configuration/rules-blockly.html#blockly-toolbox).
+
+{::options toc_levels="2..4"/}
+
+- TOC
+{:toc}
+
+{: #blockly-items-and-things-overview}
+
+## Overview of the Items and Things category
+
+> ![items-and-things](images/blockly/blockly-items-and-things.png)
+
+
+### Item and Thing Blocks
+Most of the time you will want to get and set the state of an *item* - set a switch to ON, or get a temperature. Sometimes you may want to access the *thing* directly. Both are possible using the item-block and thing-block
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.